framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2651,7 +2651,7 @@ protected void parseStubFiles() {
      * @return the annotation mirror for anno
      */
     @Override
-    public AnnotationMirror getDeclAnnotation(Element elt,
+    public final AnnotationMirror getDeclAnnotation(Element elt,
```

Comment:
An `@see` to the three-argument getDeclAnnotation would be good and a note that that version should be overridden to adapt the behavior of this method and the `-NoAliases` version.